homeassistant/components/media_player/mediaroom.py

```diff
@@ -0,0 +1,199 @@
+"""
+Support for the Mediaroom Set-up-box.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.mediaroom/
+"""
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between import groups standard library, 3rd party and homeassistant.